docs/topics/kapt.md

```diff
@@ -76,6 +76,15 @@ If you use annotation processors for your `androidTest` or `test` sources, the r
 `kaptAndroidTest` and `kaptTest`. Note that `kaptAndroidTest` and `kaptTest` extends `kapt`, so you can just provide the
 `kapt` dependency and it will be available both for production sources and tests.
 
+> To use all the newest Kotlin features with kapt, for example, [repeatable annotations](annotations.md#repeatable-annotations), 
+> add the following option to your `gradle.properties`:
```

Comment:
I'd suggest explaining why this particular option is used, like 'To use the newest Kotlin features in kapt, for example..., enable the support for the IR backend(link) with following option'

Reply:
Good idea, just one remark: I. suppose the preposition should be "with" kapt, not "in" kapt.